tsconfig.json

```diff
@@ -9,5 +10,5 @@
             "*": ["node_modules/*"]
         }
     },
-    "include": ["src/**/*"]
+    "include": ["src/main/**/*", "src/*"]
```

Comment:
we just need to compile the config.ts in the src directory na ?

we can add it as src/config.ts don't need to include the whole src directory for that

Reply:
I had thought that it would be good if we added any more files at that level.